docs/documentation/functions/aria-suite/aria-operations-networks/New-AriaNetworksvCenterDataSource.md

```diff
@@ -0,0 +1,128 @@
+# New-AriaNetworksvCenterDataSource
+
+## Synopsis
+
+Add a new vCenter Data Source to VMware Aria Operations for Networks.

```

Comment:
```suggestion
Add a new vCenter Server data source to VMware Aria Operations for Networks.

```

Reply:
Resolved in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/954c74bc51b1fef11dc26b704cc60d52dd1bba80